internal/corazawaf/rule.go

```diff
@@ -242,8 +242,17 @@ func (r *Rule) doEvaluate(logger debuglog.Logger, phase types.RulePhase, tx *Tra
 			}
 			vLog.Debug().Msg("Expanding arguments for rule")
 
+			args := make([]string, 1)
```

Comment:
it seems worth removing this allocation as well I guess?

Reply:
Yes, I have some a WIP with further refactoring, but so far I've not been able to achieve more performance / fewer allocations, even removing this line. I would merge this first to get some optimizations in and iterate over for further ones

Reply:
i see, nice try anyways!